runtime/src/runtime/fvm.rs

```diff
@@ -57,6 +57,13 @@ impl Default for FvmRuntime {
 }
 
 impl<B> FvmRuntime<B> {
+    /// Abort with "assertion failed" if the actor failed to validate the caller somewhere.
```

Comment:
This isn't wrong, but I'm not sure if it's related?

Reply:
old.